config/initializers/money.rb

```diff
@@ -3,6 +3,5 @@
 end
 
 Rails.application.config.to_prepare do
-  require 'three_scale/money'
   ActionController::Base.helper(ThreeScale::MoneyHelper) if defined?(ActionController)
```

Comment:
One more remark. Perhaps inside `to_prepare`, `ActionController` is already available so this condition could be removed?

Reply:
I'll check this in the rails 6 branch, but I prefer to ignore it here since it's not directly related to the PR